### PR TITLE
[WIP]Translate steam metadata in french

### PR DESCRIPTION
**Pull request by clfer**
Hi,

I translated Steam description and achievements to french.

It's almost complete :smile: 

I would appreciate your input on the following achievements.
For each I made some suggestions with comments.

| _**Shapeshifter**_  |  |
| --- | --- |
| **Métamorphe** | _Literal translation but not very used in french and has an organic connotation (werewolves are shapeshifter but not really mech)_ | 
| **Transformer** | _Famous transforming mechs and used as is in french (but isn't it a trademark :sweat_smile:)_ | 
|  **Pilote** |   _"Pilot" as you mastered piloting all the mechs and ships (personal suggestion)_ | 

| _**Browser**_ |  |
| --- | --- |
| **Navigateur** | _Literal translation but double meaning of "Navigator/Sailor" in French_ | 
| **Explorateur** | _"Explorer" (personal suggestion)_ | 

| _**Collector**_ |  |
| --- | --- |
| **Collecteur** | _like functional collector_ | 
| **Collectionneur** | _like trading card collector_ | 
| **Percepteur**  | _like tax collector_ | 

Thanks again for this amazing game

### CODE REVIEW

**Comment by clfer** [comment]
Hi @runiArnsbjornson ,
You can review this for steam metadata translation if you want.

Thanks

**Comment by itskatt** [comment]
I agree with your suggestions for the first two points, and for the last one, I think that *Collectionneur* will sound the best.
Otherwise, thank you very much for your hard work!

File: fastlane/metadata/steam/french/description.txt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+[*] 120+ blocs technologiques à maîtriser.
+[*] 19 types de drones, mechs et vaisseau.
+[*] 50+ succès à débloquer.
+[/list]
+
+[h2]Modes de jeu[/h2]
+
+[*] [b]Survie[/b]: Construisez des tourelles pour défendre votre base des ennemis dans un style de jeu inspiré des tower-defense. Survivez aussi longtemps que possible, avant de faire décoller votre noyau. Utiliser les ressources récoltées pour débloquer de nouvelles technologies. Préparez votre base pour les attaques de puissants boss aériens.
+[*] [b]Attaque[/b]: Construisez des usines et produisez des unités pour détruire les noyaux ennemis, tout en défendant votre base contre les vagues d'attaque. Créez une grande variété d'unités offensives et de support pour vous aider dans vos objectifs.
+[*] [b]PvP[/b]: Affrontez d'autres joueurs. Constituez jusqu'à 4 équipes différentes et tentez ed détruire les noyaux adverses. Créez des unités d'assaut ou attaquez directement les autres bases avec vos mechs.
```
Comment (Paulao17) [inline fastlane/metadata/steam/french/description.txt]: Typo here : "tentez de détruire" is correct

Comment (clfer) [reply fastlane/metadata/steam/french/description.txt]: Fixed
Thanks

**Comment by Paulao17** [comment]
Apparently `Shapeshifter` can be translated to `Transformiste`. This is related to the Theory of Evolution and suggest that the player evolved.

**Comment by itskatt** [comment]
@clfer Any updates on this?

**Comment by Anuken** [comment]
Is this complete enough to be merged?

**Comment by Paulao17** [comment]
It's good work but is now most likely outdated.

**Comment by clfer** [comment]
Hello @Anuken

I cleaned a bit and i think it's OK to be merged.

@Paulao17 I integrated your suggestion for "Shapeshifter"
feel free to review if you have any more suggestion

**Pull request merged**